pkg/restore/client.go

```diff
@@ -400,6 +400,12 @@ func (rc *Client) createTable(
 	if err != nil {
 		return CreatedTable{}, errors.Trace(err)
 	}
+	if newTableInfo.IsCommonHandle != table.Info.IsCommonHandle {
+		return CreatedTable{}, errors.Annotatef(berrors.ErrRestoreModeMismatch,
+			"CommonIsHandle: backup table %v, new table %v not match",
```

Comment:
the wording is too cryptic for users.

Reply:
🤔 How about `ClusterIndex option: backup table %v, created table %v not match`?

Reply:
> Clustered index option mismatch. Restored cluster's @@tidb_enable_clustered_index should be %[1]v (backup table = %[1]v, created table = %[2]v).